### PR TITLE
Added useful tip with local addresses

### PR DESCRIPTION
**Pull request by smileart**
Added useful tip with local addresses not to run `ifconfig` command every time you want to recall your local IP.
Once again :wink: #60

### CODE REVIEW

**Comment by indexzero** [comment]
lgtm at first glace. I will double check over the weekend and (likely) merge this in as is.

**Comment by smileart** [comment]
Looks like it's going to repeat its predecessor's fate which had awaited 1 year 3 month and 7 days to be ~~not~~ merged :grinning: :octocat:

File: bin/http-server [hunk 1]
```diff
@@ -98,6 +101,16 @@ function listen(port) {
       + ' on: '.yellow
       + (protocol + '//' + host + ':' + port).cyan);
 
+    log('\nAvailable on:'.yellow);
+
+    for (var dev in ifaces) {
```
Comment (indexzero) [inline bin/http-server]: Do not use `for in` loops.

**Comment by indexzero** [comment]
Some comments for you.

File: bin/http-server (hunk 1 above)
Comment (indexzero) [inline bin/http-server]: Use `Object.keys(ifaces).forEach(function (dev) {` instead.

File: bin/http-server [hunk 2]
```diff
@@ -98,6 +101,16 @@ function listen(port) {
...
+      ifaces[dev].forEach(function(details){
```
Comment (indexzero) [inline bin/http-server]: Conform to style here: `ifaces[dev].forEach(function (details) {`

**Comment by smileart** [comment]
Are there still any chances of merging or at least adding such a simple function? I'm just curious, nothing personal. :innocent:

**Comment by indexzero** [comment]
Whoops. Didn't get emails for your new commits. In the future comment after committing and I'll be sure to merge it faster.

**Pull request merged**